pom.xml

```diff
@@ -25,8 +25,15 @@
 		<sonar.jacoco.reportPath>../target/jacoco.exec</sonar.jacoco.reportPath>
 		<sonar.jacoco.itReportPath>../../../target/jacoco-it.exec</sonar.jacoco.itReportPath>
 		<sonar.jacoco.reportPaths>${sonar.jacoco.reportPath},${sonar.jacoco.itReportPath}</sonar.jacoco.reportPaths>
+		<!-- by default, don't run the deploy-to-jboss.org profile for all poms; set this to false in *updatesite/pom.xml -->
+		<maven.deploy.skip>true</maven.deploy.skip>
+		<skipDeployToJBossOrg>true</skipDeployToJBossOrg>
+		<skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
+		<filemgmtJbossOrg>10.5.105.197</filemgmtJbossOrg>
```

Comment:
seems weird to have an hardcoded ip, an hostname is not available? (so that if server address is modified it can still work)

Reply:
hostname IS available but it's slower than the IP (DNS resolution has been proven to be slow as it bounces through load balancers).

Reply:
ok, might worth adding the hostname in comment then. Like that if the IP is modified, future developer when investigating will find to what this IP was corresponding.

Reply:
you can deduce the domain from the property. :) Trying to keep internal details internal because they're internal. :D